cogs/shareLink.py

```diff
@@ -31,15 +31,21 @@ async def track(self, ctx: Context, *args: str):
 
     @commands.command()
     async def album(self, ctx: Context, *args: str):
-        join_words = " ".join(args)
+        join_words = " ".join(args).strip()
         results = self.sp.search(q=join_words, limit=1, type="album")
         await ctx.channel.send(
             results["albums"]["items"][0]["external_urls"]["spotify"]
         )
 
+    @classmethod
+    def is_enabled(cls, configs: Dict[str, Any] = {}):
+        return (
+            configs["ENABLE_SHARE_LINK"]
```

Comment:
Add config bools for if the client id's and stuff exist

Reply:
should be fixed